src/schema-gen/InspectorXSLT/testing/XSD-VALIDATE-COMPUTER-SAMPLES.xpl

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc" xmlns:c="http://www.w3.org/ns/xproc-step"
+   xmlns:cx="http://xmlcalabash.com/ns/extensions" version="1.0"
+   xmlns:metaschema="http://csrc.nist.gov/ns/metaschema/1.0" type="metaschema:XSD-FUNCTIONAL-SAMPLES"
+   name="XSD-FUNCTIONAL-SAMPLES" xmlns:x="http://www.jenitennison.com/xslt/xspec"
+   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:nm="http://csrc.nist.gov/ns/metaschema"
+   xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+   <!-- Input: depends on finding file 'inspector-functional-xspec/validations-in-batch.xspec' in place -->
+   <!-- Input: Additionally, all inputs named therein (expect errors for files broken or missing)-->
+   <!-- Input: Additionally, an up-to-date XSD for the computer model located at '../current/computer_metaschema-schema.xsd' -->
+   <!-- Output: an 'all is well' message, or unexpected results such as errors from files expected to be valid or validity from files expected to be invalid -->
+   <!-- Purpose: test alignment between XSD-based Metaschea validates and other forms, by providing a basis for comparisohn-->
+
+
+   <!-- &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& &&& -->
+   <!-- Ports -->
+
+   <p:input port="parameters" kind="parameter"/>
+
+
+   <!-- Lists are being maintained by hand for now, for clarity and robustness -->
+   <!-- Align with file 'inspector-functional-xspec/validations-in-batch.xspec'-->
+
+   <!-- TODO: separate out layers
+     provide valid and invalid tags as options
+     (along with file list and XSD)
+     parameterizing them for XSLT -->
+
+   <p:input port="samples" sequence="true">
+      <p:document href="computer-valid/valid1.xml"/>
+      <p:document href="computer-valid/valid2.xml"/>
+
+      <p:document href="computer-invalid/invalid1.xml"/>
+      <p:document href="computer-invalid/invalid2.xml"/>
+      <p:document href="computer-invalid/invalid3.xml"/>
+      <p:document href="computer-invalid/invalid4.xml"/>
+      <p:document href="computer-invalid/invalid5.xml"/>
+      <p:document href="computer-invalid/invalid6.xml"/>
+      <p:document href="computer-invalid/invalid7.xml"/>
+      <p:document href="computer-invalid/invalid8.xml"/>
+      <p:document href="computer-invalid/invalid9.xml"/>
+      <p:document href="computer-invalid/invalid10.xml"/>
+   </p:input>
+
+   <p:input port="computer-schema">
+      <p:document href="current/computer_metaschema-schema.xsd"/>
```

Comment:
Why not use a generated version on the fly and not check it into source yet if we have this XProc, this script, and other scripts for generation and have it in a directory like `test_output` not checked into the git repo?